lib/std/Build.zig

```diff
@@ -401,6 +408,8 @@ fn createChildOnly(parent: *Build, dep_name: []const u8, build_root: Cache.Direc
         .modules = std.StringArrayHashMap(*Module).init(allocator),
         .initialized_deps = parent.initialized_deps,
         .available_deps = pkg_deps,
+        .compdb_entries_dir = parent.compdb_entries_dir,
+        .compdb_entry_paths_set = parent.compdb_entry_paths_set,
```

Comment:
This will not really work out as a `.put` might mutate the set. Make `compdb_entry_paths_set` a `?*StringHashMap(void) = null` and move the storage and generation of the string hash map into the `build_runner.zig` file.

This way no builder will have a copy/clone of the map but just reference to a global object

Reply:
Apart from that i'm happy now :blush: 
Dis going to be great!

Reply:
@MasterQ32 I just made the updates necessary.